SRS.md

```diff
@@ -1,28 +1,98 @@
-
-
-By clicking green button user will add new employee. To add new employee user should write down his department, some personal data and his salary.
-Then user should press green button "save" to store this data in database.  
-
-![image](https://user-images.githubusercontent.com/83345134/127285174-8b015c12-a1b2-4ff9-a32c-d93a516a4bbb.png "Add employee")
-
-To edit enployee's information the user should double click employee in the table, fill in the required info and press "Yes", otherwise press "Cancel" to return back: 
-
-![image](https://user-images.githubusercontent.com/83345134/127285373-3f14cb74-9510-4dc3-8e3a-2c2c01dee4a4.png "Edit employee")
-
-To delete an employee user should right-click eployee in the table and press delete button. Otherwise, hi will need to press "Cancel" button to return back to employees' list:
-
-![image](https://user-images.githubusercontent.com/83345134/127285318-fc4847d6-ff49-4f51-863a-19ffafd29391.png "Delete employee")
+  __Constraints for data validation:__  
+  * Department – one of the departments;  
+  * Name –  maximum length of 45 characters;  
+  * Salary – maximum value is 2000;  
+  * Date of birth – date in format dd/mm/yyyy.   
+      
+  ### 2.3 Edit employee  
+  __Main scenario:__  
+  * User clicks the “Edit” button in the employees list view mode;  
+  * Application displays form to enter employee data;  
+  * User enters cemployee’s data and presses “Save” button;  
+  * If any data is entered incorrectly, incorrect data messages are displayed;  
+  * If entered data is valid, then edited data is added to database;  
+  * If error occurs, then error message is displaying;  
+  * If employee’s record is successfully edited, then list of employees with added records is displaying.  
+ 
+  __Cancel operation scenario:__  
+  * User clicks the “Edit” button in the employees list view mode;  
+  * Application displays form to enter employee data;  
+  * User enters employee data and presses “Cancel” button;  
+  * Data don’t save in database, then list of employees records is displaying to user.  
+  * If the user selects the menu item "Departments” or "Employees", the data will not be saved to the database and the corresponding form with updated data will be opened.  
+      
+![image](https://user-images.githubusercontent.com/83345134/127285373-3f14cb74-9510-4dc3-8e3a-2c2c01dee4a4.png "Edit employee")  
+    Pic. 2.3 Edit employee.    
+    
+  ### 2.4 Removing employee  
+  __Main scenario:__  
+  * The user, while in the list of employees mode, presses the "Delete" button in the selected employee line;  
+  * Application displays confirmation dialog “Please confirm delete employee?”;  
+  * The user confirms the removal of the employee;  
+  * Record is deleted from database;  
+  * If error occurs, then error message displays;  
+  * If employee record is successfully deleted, then list of employees without deleted records is displaying.  
+      
+  __Cancel operation scenario:__  
```

Comment:
I think this scenario should be described in a more general way, as it applies to all actions: "Add", "Edit" and "Delete".